exterminate.lua

```diff
@@ -181,6 +181,33 @@ if positionals[1]:lower() == 'undead' then
             count = count + 1
         end
     end
+elseif positionals[1]:split(':')[1] == "all" then
+    local selected_caste = nil
+  
+    if string.find(positionals[1], ':') then
+        local tokens = positionals[1]:split(':')
+        selected_caste = tokens[2]
+    end
```

Comment:
```suggestion
    local selected_caste = positionals[1]:split(':')[2]
```

Reply:
I realize this was copied from the section below, but since you don't care about the "race" (`all`) here, it can be simplified